src/components/Tab/Tab.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const Tab = (props) => {
+  const {
+    handleClick, title, activeTab, id,
```

Comment:
will be better for Tab to receive prop isActive instead of activeTab

Reply:
will be better to name prop onClick instead of handleClick here, but name this function handleClick in parent component